bin/journey-test.sh

```diff
@@ -305,7 +304,6 @@ main() {
   local trackler_home="${build_path}/trackler"
   local exercism_home="${build_path}/exercism"
 
-  local exercism_configfile=".journey-test.exercism.json"
```

Comment:
Does anyone know anything about this file? I can't find it anywhere so I'm not sure if it's important or not...